manifest.toml

```diff
@@ -19,7 +19,7 @@ admindoc = "https://freshrss.github.io/FreshRSS/"
 code = "https://github.com/FreshRSS/FreshRSS"
 
 [integration]
-yunohost = ">= 11.2.30"
+yunohost = ">= 12.0.9"
```

Comment:
Suggestion: same as below for the update from PHP 8.3 to 8.4. I think it is best to perform such changes of major requirements during a new version such as FreshRSS 1.26.0 or FreshRSS 1.27.0 and not during a patch version. This will also block potential remaining users of YunoHost 11 on a version of FreshRSS with known security vulnerabilities.